scalatest/src/main/scala/org/scalatest/Suite.scala

```diff
@@ -756,7 +756,7 @@ trait Suite extends Assertions with Serializable { thisSuite =>
    *     exists in this <code>Suite</code>
    */
   final def execute(
-    testName: String = null,
+    testName: String | Null = null,
```

Comment:
**Nullable parameter**
Parameter that can be null and should therefore have a `| Null` type.